ddcz/views.py

```diff
@@ -346,17 +346,23 @@ def author_detail(request, author_id, slug):
 
 
 def phorum(request):
-    if request.method == "POST":
-        form = PhorumCommentForm(request.POST)
-        if form.is_valid() and request.user:
-            Phorum.objects.create(
-                reputace=0,
-                reg=1,
-                user=request.user.profile,
-                text=form.cleaned_data["text"],
-                nickname=request.user.profile.nick_uzivatele,
-                email=request.user.profile.email_uzivatele,
-            )
+    if request.method == "POST" and request.POST["post_type"] and request.user:
```

Comment:
This works, it just ocurred to me that if you'd want to avoid additional hidden field, you can inspect the value of submit button. 

But this works.